### PR TITLE
Recorded sample tests for compute

### PR DESCRIPTION
**Pull request by anudeepsharma**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@anudeepsharma__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (ans). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: Samples/ResourceManagement/Compute/ConvertVirtualMachineToManagedDisks/ConvertVirtualMachineToManagedDisks.xproj [hunk 1]
```diff
@@ -4,18 +4,16 @@
     <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">14.0</VisualStudioVersion>
     <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
   </PropertyGroup>
-
   <Import Project="$(VSToolsPath)\DotNet\Microsoft.DotNet.Props" Condition="'$(VSToolsPath)' != ''" />
   <PropertyGroup Label="Globals">
-    <ProjectGuid>e52471d0-9127-483d-a326-5ce3b6b3aa86</ProjectGuid>
+    <ProjectGuid>d8d2bd1c-f9ef-405b-ab66-0cbb3fe6e559</ProjectGuid>
     <RootNamespace>MigrateVirtualMachine</RootNamespace>
```
Comment (anuchandy) [inline Samples/ResourceManagement/Compute/ConvertVirtualMachineToManagedDisks/ConvertVirtualMachineToManagedDisks.xproj]: Confirming - shouldn't this be `<RootNamespace>ConvertVirtualMachineToManagedDisks</RootNamespace>`  ?

**Review by anuchandy — APPROVED** [review]
LGTM, one comment, please take a look

File: Samples/ResourceManagement/Compute/ConvertVirtualMachineToManagedDisks/ConvertVirtualMachineToManagedDisks.xproj (hunk 1 above)
Comment (anudeepsharma) [inline Samples/ResourceManagement/Compute/ConvertVirtualMachineToManagedDisks/ConvertVirtualMachineToManagedDisks.xproj]: yes, fixing it

**Comment by hovsepm** [comment]
+1 to LGTM

**Pull request merged**